variables.tf

```diff
@@ -60,12 +71,12 @@ variable "policy_bindings" {
 
 variable "module_enabled" {
   type        = bool
-  description = "(Optional) Whether to create resources within the module or not. Default is 'true'."
+  description = "(Optional) Whether or not to create resources within the module."
```

Comment:
```suggestion
  description = "(Optional) Toggle resource creation within the module"
```

Reply:
Has been changed to
```suggestion
  description = "(Optional) Toggle resource creation within the module."
```

Reply:
@switchdk remember to backport to the template.